packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -94,7 +95,7 @@ export const useSchedule = ({
     routedTeamMemberIds,
     skipContactOwner,
     _shouldServeCache,
-    routingFormResponseId,
+    ...(queuedFormResponseId ? { queuedFormResponseId } : { routingFormResponseId }),
```

Comment:
If `queuedFormResponseId` is there we don't set `routingFormResponseId` as that would cause the trpc request to be sent again, instead we keep on using queuedFormResponseId even if `routingFormResponseId` becomes available later.